### DISCUSSION TITLE
Undefined variable $unreadNotificationsCount

### DISCUSSION THREAD

**Discussion by ucosatti** [filamentphp/filament#11352] [Help]
### Package

Notifications

### Package Version

3.2.30

### How can we help you?

Hi! I need some help adding database-notification on a Livewire project.

I'd create the view for the trigger button and included at the app. layout 
then in AppProvider add the DatabaseNotification::trigger to the trigger view.
but I get Undefined variable $unreadNotificationsCount. (all steps in the docs)

what am I missing??

**Comment by ucosatti** [comment]
found how to. closed

**Reply by alexandrutoma6** [reply]
how?

**Reply by bernhardh** [reply]
How?

**Reply by leandrocfe** [reply]
Did you create this file? `resources/views/notifications-trigger.blade.php`

**Reply by eric-hansen** [reply]
I'm not sure if anyone will find this thread, but the answer is to follow the docs on using database notifications outside of a panel.  Then just use `<livewire:database-notifications />` (or `@livewire('database-notifications')`).  **Do not** try to load your view directly, the Livewire declaration will do that for you due to the `DatabaseNotifications::trigger()` in the provider.  That should resolve any issues around this.

**Reply by Ryuuji159** [reply]
the docs aren't very clear about that, it suggest to add the `@livewire('database-notification')` to the layout just like the `@livewire('notifications')` component

**Comment by ucosatti** [comment]
close
